bluemira/builders/EUDEMO/reactor.py

```diff
@@ -78,13 +77,14 @@ def run_systems_code(self):
         # run_systems_code interface is updated to have a more general runmode value.
         config["process_mode"] = config.pop("runmode")
 
-        output: ParameterFrame = run_systems_code(
+        self.syscode_solver = systems_code_solver(
             self._params,
             config,
             self._file_manager.generated_data_dirs["systems_code"],
             self._file_manager.reference_data_dirs["systems_code"],
         )
-        self._params.update_kw_parameters(output.to_dict())
+        self.syscode_solver.run()
+        self._params.update_kw_parameters(self.syscode_solver.params.to_dict())
```

Comment:
So here for example, if I want to see the radial build of a reactor, I have to do:

```python
import os
from bluemira.codes.wrapper import plot_radial_build
sys_code_dir = reactor.file_manager.generated_data_dirs["systems_code"]
plot_radial_build(os.sep.join([sys_code_dir, "OUT.DAT"]), width=1.0)
```
I'm not sure that's right, but if we could have a neat interface to this somehow from the `Reactor` object that'd be great.

Reply:
Ok with the most recent push it would now look like 

```python
from bluemira.codes import plot_radial_build
plot_radial_build(reactor.file_manager.generated_data_dirs["systems_code"], width=1.0)
```

which is better. But it doesnt address the original point. Working on how best to formulate a response :slightly_smiling_face:

Reply:
@DanShort12 and I had a chat about this too, off the back of #707. We should catch up and have a chat; Dan has an idea how to generalise this kind of "analysis stage" for a reactor, and access it in the same way we access build stages.

Reply:
Yea, for builders we do `register_builder` and `get_builder`, so we could do the same for solvers (or generalise the register/get for builders to something like register/get build_stage and include solvers in that). As a more future-looking change we probably also want to (perhaps optionally) store history information for builders/solvers - like we do for parameters. Then we could say 'for my ith PROCESS run, what did my radial build look like'. It will be pretty memory intensive though, hence why it's probably an option to run when developing/investigating something.